### PR TITLE
Update named_tuple.cr (typo)

### PR DESCRIPTION
**Pull request by cserb**
typo

### CODE REVIEW

File: src/named_tuple.cr [hunk 1]
```diff
@@ -26,7 +26,7 @@ struct NamedTuple
   # Creates a named tuple that will contain the given arguments.
   #
   # This method is useful in macros and generic code because with it you can
-  # creates empty named tuples, something that you can't do with a tuple literal.
+  # create empty named tuples, something that you can't do with a tuple literal.
```
Comment (jhass) [inline src/named_tuple.cr]: Maybe a good opportunity to rephrase slight? :)

> With a named tuple literal you cannot create an empty named tuple. This method doesn't have this limitation, which makes it especially useful in macros and generic code.

Comment (cserb) [reply src/named_tuple.cr]: I also thought about rephrasing, but couldn't come up with a concise solution. Yours is not better than the original one imho :grin:

Comment (cserb) [reply src/named_tuple.cr]: Sorry, I should have mentioned, that just rephrasing doesn't add much value for me. I would have preferred adding more context

Comment (straight-shoota) [reply src/named_tuple.cr]: > *options* can be empty, so this method can create an empty `NamedTuple`. A tuple literal can't express this.

Comment (cserb) [reply src/named_tuple.cr]: I took @jhass rephrasing and changed the order of the examples to emphasize the method's use-case. Have look

**Comment by cserb** [comment]
> Please leave the order of examples as it was before.

done

**Pull request merged**